mapit_global/management/commands/mapit_global_update_countries.py

```diff
@@ -84,30 +84,20 @@ def get_enclosing_country_codes(self, area):
                 enclosing_country_codes.add(country_code_object.code)
         return tuple(sorted(enclosing_country_codes))
 
-    def get_country_from_codes(self, country_codes):
+    def get_countries_from_codes(self, country_codes):
         if not country_codes:
             return None
-        if len(country_codes) > 1:
-            unique_country_code = '?'
-        else:
-            unique_country_code = next(iter(country_codes))
-        return self.code_to_country[unique_country_code]
+        return [self.code_to_country[c] for c in country_codes]
 
     def set_country_on_all_areas(self):
-        error_messages = []
+        global_country = Country.objects.get(code='G')
         for area in Area.objects.filter(**self.generation_kwargs):
             print("Considering area:", area, area.id)
             enclosing_country_codes = self.get_enclosing_country_codes(area)
-            country = self.get_country_from_codes(enclosing_country_codes)
-            if country and country.code == '?':
-                msg = 'Multiple enclosing country codes {codes} found for area {name} with ID {id}'
-                error_messages.append(msg.format(
-                    codes=unicode(enclosing_country_codes),
-                    name=area.name,
-                    id=area.id))
-            area.country = country
+            countries = self.get_countries_from_codes(enclosing_country_codes)
+            area.country = global_country
+            area.countries.add(*countries)
```

Comment:
I think this should have a `area.countries.clear()` before the `.add` if it's going to work properly on updating, in the case where a country boundary used to but no longer encloses one of the arbitrary points in the area.

Another thing is that `countries` could be `None`, and this line would throw an exception in that case. I think that's better dealt with by removing these two lines:
```
        if not country_codes:
            return None
```
... from `get_countries_from_codes` though.